go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/longhorn/longhorn-engine
 
 go 1.13
 
+replace github.com/longhorn/sparse-tools => /Users/peterle/go/src/github.com/longhorn/sparse-tools
```

Comment:
I merge the sparse tools PR https://github.com/longhorn/sparse-tools/pull/77 thanks

Reply:
Thanks